src/dbnode/storage/namespace/map_test.go

```diff
@@ -24,7 +24,6 @@ import (
 	"testing"
 
 	"github.com/m3db/m3x/ident"
-
```

Comment:
Did you remove this newline? I think its required to prevent the linter from complaining (we separate our dependencies from others)

Reply:
not sure, but will follow the imports conventions as you mentioned below